scripts/reboot

```diff
@@ -265,9 +280,13 @@ if [ -x ${DEVPATH}/${PLATFORM}/${PRE_REBOOT_HOOK} ]; then
     fi
 fi
 
-if [ -x ${WATCHDOG_UTIL} ]; then
+is_platform_support_180s_wd
+
+if [[ -x ${WATCHDOG_UTIL} && "${IS_PLATFORM_SUPPORT_180S_WD}" == 1 ]]; then
     debug "Enabling the Watchdog before reboot"
     ${WATCHDOG_UTIL} arm
+else
+    debug "Skip enabling the watchdog"
```

Comment:
There are downsides to blanket disable watchdog. Especially in the case of warm-reboot and such time sensitive reboots.